data/src/main/java/com/example/data/di/NetworkModule.kt

```diff
@@ -86,4 +87,10 @@ object NetworkModule {
     fun provideFollowerService(retrofit: Retrofit): UserService {
         return retrofit.create(UserService::class.java)
     }
+
+    @Singleton
+    @Provides
+    fun provideSearchService(retrofit: Retrofit): SearchService {
+        return retrofit.create(SearchService::class.java)
```

Comment:
```suggestion
        return retrofit.create(SearchService::class.java)
```
refield를 사용해서 Class 타입을 매개변수로 넘기지 않는 방식을 찾아봐도 좋을 것 같아용!

Reply:
@jaeryo2357 와.. 이런 방법도 있군용 ㄷㄷ 감사합니다!!